sbf-parser-fuzz/src/main.rs

```diff
@@ -0,0 +1,32 @@
+use afl::*;
+
+use libsbf::Messages;
+use libsbf::parser::SbfParser;
+
+fn main() {
+    fuzz!(|data: &[u8]|{
+        let mut parser = SbfParser::new();
+        let mut remaining = data;
+        
+        while !remaining.is_empty() {
+            match parser.consume(remaining) {
+                Ok((Some(message), bytes_consumed)) => {
+                    eprintln!("got msg");
+                    if let Messages::Unsupported = message {
+                        panic!("We should never get an Unsupported message from the parser");
```

Comment:
What if it thinks it read a supported message?

Reply:
What do you mean?

Reply:
I was thinking of a case where the fuzzer injects bytes in the middle and it still thinks it's a valid message but the crc should fail in that case so nvm.